packages/fxa-content-server/tests/server/amplitude.js

```diff
@@ -740,7 +740,7 @@ registerSuite('amplitude', {
         'fxa_pref - account_recovery_option_download'
       );
     },
-
+    // TODO remove in FXA-7419, print option not available in new UI
```

Comment:
Thanks for adding this note here too! 🙌 

I don't think we've added test cases here for our new events - that might be worthwhile addition?

Reply:
wunderbar!!!